src/fmu/sumo/explorer/_child_object.py

```diff
@@ -27,22 +41,58 @@ def __init__(self, sumo_client, meta_data):
 
     @property
     def blob(self):
+        """Returns the __blob attribute"""
         if self.__blob is None:
             self.__blob = self.__get_blob()
 
         return self.__blob
 
     def __get_blob(self):
+        """Gets the object from blob store"""
         blob = self.sumo.get(f"/objects('{self.sumo_id}')/blob")
         return blob
 
     @property
     def png(self):
+        """returns the __png attribute"""
         if self.__png is None:
             self.__png = self.__get_png()
 
         return self.__png
 
     def __get_png(self):
+        """Gets the object as png"""
         png = self.sumo.get(f"/objects('{self.sumo_id}')/blob", encoder="png")
-        return png
\ No newline at end of file
+        return png
+
+
+class SurfaceObjects:
+
+    """Container for a set of objects"""
+
+    def __init__(object_ids, case):
+        """Init of objects
+        args:
+        object_ids (dict): key is real, value is object id
+        case fmu.sumo.Ex
+        """
+        self._object_ids = object_ids
+        self._sumo = case
+
+    @property
+    def object_ids(self):
```

Comment:
Related to the previous comment, but first glance this seems a bit strange. `object_ids` is really a dict of realizations, where object_id is (the only?) key. This scales a bit poorly as well. E.g. what if other attributes are needed. An option could be to flip it, and use object_id as key, and realization_id as (one) of the values. Then it can be expanded to more values if needed.

Reply:
See previous conversation.